ooni/nettests/blocking/dns_consistency.py

```diff
@@ -182,12 +189,14 @@ def lookup_details():
                     log.msg("Further testing has eliminated false positives")
                     lookup_details()
                     log.msg("tampering: reverse_match")
-                    self.report['tampering'][test_resolver] = 'reverse_match'
+                    self.report['errors'][test_resolver] = 'reverse_match'
+                    self.report['successful'].append(test_resolver)
                 else:
                     log.msg("Reverse lookups do not match")
                     lookup_details()
                     log.msg("tampering: true")
-                    self.report['tampering'][test_resolver] = True
+                    self.report['errors'][test_resolver] = True
+                    self.report['inconsistent'].append(test_resolver)
 
     def inputProcessor(self, filename=None):
         """
```

Comment:
Correct.

You can see the diff for this on the ooni-spec PR for the data format 0.2: https://github.com/TheTorProject/ooni-spec/pull/49